support/package-aur/manager.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+dir=support/package-aur
+if [[ -d ../../${dir} ]]; then
+  cd ../..
+elif [[ ! -d ${dir} ]]; then
+  echo "this script should be executed from project root dir."
+  exit 1
+fi
+
+function fn_setup {
+  pkgname=redshape
+  pkgver=$(grep 'version' package.json | sed -E "s| *\"version\" *: *\"([^\"]+)\".*|\1|")
+}
+
+function fn_pack {
+  output=${dir}/${pkgname}-${pkgver}.tar.gz
+  tar cvfz ./$output --exclude='./.circleci' --exclude='__tests__' --exclude='__mocks__' --exclude='./coverage' --exclude='./support' --exclude='./dist' --exclude='./test' --exclude='./docs' --exclude='./node_modules' --exclude='./build' --exclude-vcs --exclude-vcs-ignores . --transform='s|^\.|'"${pkgname}-${pkgver}"'|' 
+  echo "packed in ${output}"
+}
+
+function fn_pkgbuild {
+  output=${dir}/${pkgname}-${pkgver}.tar.gz
+  md5=$(md5sum ${output} | tr -s ' ' | cut -d' ' -f1) 
+  sed -i -E 's|pkgver=.*$|pkgver='"$pkgver"'|;s|md5sums=.*$|md5sums=("'"${md5}"'")|' ${dir}/PKGBUILD
+  echo "updated ${dir}/PKGBUILD with version ${pkgver} and checksum ${md5}"
+}
+
+function fn_makepkg {
```

Comment:
I assume that you built the image locally on your machine, right?
For now build for win and ubuntu is done automatically on merge to master. This is done using the electron-builder package, which automatically upload the binaries to the /releases section. It also handles auto-update once the application is started.
For macOS, however builds are done locally cause it needs to be signed and it can only be done from an image / machine, running on macOS. I played with macOS machines that circleCi has, but it didn't work with the keychaining and key generation.

What about this? Will it support auto-update feature? Can it be run automatically from CircleCI (I mean, it can, but will it require any different or specific steps, ot it'll work following the electron-builder's logic?)